docs/getting_started.rst

```diff
@@ -0,0 +1,80 @@
+=======================================
+Installing Drydock in a Dev Environment
+=======================================
+
+Drydock runs in Python 3.x only and is tested on Ubuntu 16.04 standard
+images. It is recommended that your development environment be a Ubuntu
+16.04 virtual machine.
+
+MaaS
+----
+
+Drydock requires a downstream node provisioning service and currently
+the only driver implemented is for Canonical MaaS. So to begin with
+install MaaS following their instructions_ https://docs.ubuntu.com/maas/2.2/en/installconfig-package-install. 
+The MaaS region and rack controllers can be installed in the same VM 
+as Drydock or a separate VM.
+
+On the VM that MaaS is installed on, create an admin user:
+
+::
+
+    $ sudo maas createadmin --username=admin --email=admin@example.com
+
+You can now access the MaaS UI by pointing a browser at http://maas_vm_ip:5240/MAAS
+and follow the configuration journey_ https://docs.ubuntu.com/maas/2.2/en/installconfig-webui-conf-journey
+to finish getting MaaS ready for use.
+
+Drydock Configuration
+---------------------
+
+Clone the git repo and customize your configuration file
+
+::
+
+    git clone https://github.com/att-comdev/drydock
+    cd drydock
+    tox -e genconfig 
+    cp -r etc /tmp/drydock-etc
+
+In `/tmp/drydock-etc/drydock/drydock.conf` customize your maas_api_url to be
```

Comment:
This is just a way to run Drydock w/o impacting your system. It doesn't really matter where it resides as it gets mounted into the container as /etc/drydock.